docker/README.md

```diff
@@ -0,0 +1,53 @@
+# bepasty Docker image
+
+A Docker image that provides very basic configuration via environment variables.
+
+**Notes**
+
+* For a more advanced configuration mount a custom configuration to
+  `/etc/bepasty.conf`.
+* All data will be owned by UID 117356, to change this you can use the
+  `--user <uid>` option from `docker run`.
+
+## Quickstart
+
+```sh
+# build from the repository root
+docker build -f docker/Dockerfile -t bepasty .
+
+# create datadir
+mkdir data
+sudo chown 17357:0 data
```

Comment:
the chown uid is not the same as the uid the comment (line 9) talks about.

also unclear to me: where does that uid come from?

Reply:
thanks! fixed